.github/ISSUE_TEMPLATE/project_spec.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
This is to replace us manually creating blank issues and have a standardized format going forward. This was inspired by the Grafana K6 app repo.